src/libs/actions/BankAccounts.ts

```diff
@@ -174,103 +192,90 @@ function addPersonalBankAccount(account) {
     API.write(commandName, parameters, onyxData);
 }
 
-function deletePaymentBankAccount(bankAccountID) {
-    API.write(
-        'DeletePaymentBankAccount',
-        {
-            bankAccountID,
-        },
-        {
-            optimisticData: [
-                {
-                    onyxMethod: Onyx.METHOD.MERGE,
-                    key: `${ONYXKEYS.BANK_ACCOUNT_LIST}`,
-                    value: {[bankAccountID]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}},
-                },
-            ],
-
-            // Sometimes pusher updates aren't received when we close the App while still offline,
-            // so we are setting the bankAccount to null here to ensure that it gets cleared out once we come back online.
-            successData: [
-                {
-                    onyxMethod: Onyx.METHOD.MERGE,
-                    key: `${ONYXKEYS.BANK_ACCOUNT_LIST}`,
-                    value: {[bankAccountID]: null},
-                },
-            ],
-        },
-    );
+function deletePaymentBankAccount(bankAccountID: number) {
+    type DeletePaymentBankAccountParams = {bankAccountID: number};
```

Comment:
Same - OK, if I see any more, I'm going to skip commenting on them.

Reply:
Ah, I see now that maybe this is the pattern to follow for params to API calls?

Reply:
I wonder if there is a good reason for still not wanting to define them in their own files... 🤔

Reply:
@tgolen Yeah, we agreed to define API params types like this for now. As I know, we are going to improve API typing in the future, so it can be put in a separate file then
^^ @blazejkustra @kubabutkiewicz

Reply:
Yes @tgolen there was a message from @fabioh8010 that we will follow this for now 
<img width="1334" alt="image" src="https://github.com/Expensify/App/assets/25230417/39976d0f-e957-42c0-b726-44a3aa92f1ab">

Reply:
I'm totally on board with that approach. Thanks for making me aware of it!